avidemux_ru.ts

```diff
@@ -12842,7 +12842,7 @@ Ready.
     <message>
         <location line="+1"/>
         <source>swScale Resizer.</source>
-        <translation>Изменить размер посредством libswscale.</translation>
+        <translation>Изменить размер с помощью swScale.</translation>
```

Comment:
I admit that I didn't dare to fix the original source, but the library is called libswscale while functions from this library are often prefixed with "swscale_" or "sws_". "swScale" is not a thing.

Reply:
Fixed.